dist/index.html

```diff
@@ -236,49 +218,70 @@ <h3 class="section-subheading text-muted">A part by part breakdown of how we mak
             </div>
         </div>
     </section>
-    <!-- Team-->
-    <section class="page-section bg-light" id="team">
+
+
+
+
+
+
+    <!-- Order section-->
+    <section class="page-section bg-light" id="Order">
         <div class="container">
             <div class="text-center">
-                <h2 class="section-heading text-uppercase">Our Team</h2>
-                <h3 class="section-subheading text-muted">(It's a small one.).</h3>
+                <h2 class="section-heading text-uppercase">Order a Bullwhip</h2>
+                <h3 class="section-subheading text-muted">Order a custom bullwhip, a whip in stock, kelvar crackers, or request repair services.</h3>
             </div>
             <div class="row">
-                <div class="col-lg-4">
-                    <div class="team-member">
-                        <img class="mx-auto rounded-circle" src="assets/img/Gerrit.jpg" alt="" />
-                        <h4>Gerrit Butler</h4>
-                        <p class="text-muted">Whipmaker</p>
-                        <a class="btn btn-dark btn-social mx-2" href="https://twitter.com/GerritsWhips"><i class="fab fa-twitter"></i></a>
-                        <a class="btn btn-dark btn-social mx-2" href="https://www.facebook.com/GerritsWhipFactory"><i class="fab fa-facebook-f"></i></a>
-                        <a class="btn btn-dark btn-social mx-2" href="https://www.instagram.com/gerritswhips/"><i class="fab fa-instagram"></i></a>
+                <div class="col-lg-4 col-sm-6 mb-4">
+                    <div class="portfolio-item">
```

Comment:
should be "order-item" to reflect new .scss file

Reply:
there are too many instances to comment of this but anytime in this section that it says portfolio it should be "order" to reflect new scss file